### DISCUSSION TITLE
Markdown lesson

### DISCUSSION THREAD

**Discussion by CBID2** [TheOdinProject/curriculum#25756] [Ideas]
I was reading through [this discussion](https://github.com/TheOdinProject/curriculum/discussions/25472) and decided to take a step further and suggest a lesson on Markdown. Developers are going to encounter documentation, and Markdown is commonly used to create this document. So, I think having this course would benefit them in the long run.

**Comment by CBID2** [comment]
Hey @CouchofTomato! :) I'd love to hear your thoughts on this.

**Comment by CouchofTomato** [comment]
Hey @CBID2 

I'm not sure whether this would enhance the curriculum or not. Markdown is pretty easy to pick up. I think it would be better to maybe link to the github documentation on writing markdown around the time we tell learners to create their first repo.

**Reply by CBID2** [reply]
Thank you for your response @CouchofTomato! :)  How about this? Have the lesson focus on certain Markdown syntax and formatting rules to consider, link to another source like you recommend, and then have learners a create ReadMe file as an assignment. That way they can apply the knowledge they have learned.

**Comment by CouchofTomato** [comment]
Is that worth an entire lesson? I'm not sure of the benefit. I haven't seen any learners have many issues with markdown to be honest. Have you seen any comments?

**Reply by CBID2** [reply]
> Is that worth an entire lesson? I'm not sure of the benefit. I haven't seen any learners have many issues with markdown to be honest. Have you seen any comments?

I brought the idea up in Discord. They seem to like it :)

**Reply by CouchofTomato** [reply]
Can you link to that please.

**Reply by CBID2** [reply]
Sure

**Reply by CBID2** [reply]
> Can you link to that please.

Here you go [request for markdown](https://discord.com/channels/505093832157691914/505093832157691916/1124755764905529438)

**Reply by CouchofTomato** [reply]
Thanks. I'm not convinced we need an entire lesson dedicated to this. Part of learning is going off and exploring things on your own, My opinion remains that we could link to the github markdown docs before we ask learners to write their first readme for a project.

**Reply by CBID2** [reply]
Ok deal
